### PR TITLE
added current_developer & developer_signed_in?

### PR DESCRIPTION
**Pull request by adamggg**
@NourhanMohamed <- Review code
@zoheiry <- Review documentation

### CODE REVIEW

**Comment by adamggg** [comment]
Bare in mind I haven't attached any UMLs for it.

**Comment by NourhanMohamed** [comment]
Ghanem said we can't merge anything if it wasn't completely verified
and you should tag the issue number up there in the pull request :)

File: arability/app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+  # sure you write "include ApplicationHelper" at the top of the file
+
+  # Checks if current signed in gamer has a developer account
+  # author:
+  #   Adam Ghanem
+  # params:
+  #   none
+  # returns:
+  #   true if the gamer has a developer account otherwise false
+  def developer_signed_in?
```
Comment (NourhanMohamed) [inline arability/app/helpers/application_helper.rb:14]: indent both methods to be 2 spaces from the hashes of the documentation "same level as author and params", we're used to that convention

File: arability/app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+  #   Adam Ghanem
+  # params:
+  #   none
+  # returns:
+  #   success: returns the current developer that is signed in
+  #   failure: returns nil
+  def current_developer
+    Developer.find_by_gamer_id(current_gamer.id)
+  end
+
```
Comment (NourhanMohamed) [inline arability/app/helpers/application_helper.rb:29]: remove this empty line

File: arability/app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,30 @@
 module ApplicationHelper
+
```
Comment (NourhanMohamed) [inline arability/app/helpers/application_helper.rb:2]: remove this empty line

**Comment by NourhanMohamed** [comment]
Few changes and the code will be verified

**Pull request merged**